core/src/main/scala/com/salesforce/op/stages/impl/feature/OpScalarStandardScaler.scala

```diff
@@ -60,9 +60,17 @@ class OpScalarStandardScaler
     val internalScaler = new MLLibStandardScaler(withMean = estimator.getWithMean, withStd = estimator.getWithStd)
     val scalerModel = internalScaler.fit(vecData)
 
+    val std = scalerModel.std.toArray
+    val mean = scalerModel.mean.toArray
+    val stdMean = std.sum / std.length
```

Comment:
This sounds like a good idea. Will do.

Reply:
Actually, would it be better to only have the metadata be set if the input "vector" is length 1?  If it is uses a `require` statement it would not allow people to input multiple vectors.